src/features/Send/useCases/ListSelectedTokens/EditAmount/DeleteAmountWhenZeroed.ts

```diff
@@ -0,0 +1,26 @@
+import {useNavigation} from '@react-navigation/native'
+
+import {TxHistoryRouteNavigation} from '../../../../../navigation'
+import {useSelectedWallet} from '../../../../../SelectedWallet/Context/SelectedWalletContext'
+import {useTokenInfo} from '../../../../../yoroi-wallets/hooks'
+import {asQuantity, Quantities} from '../../../../../yoroi-wallets/utils/utils'
+import {useSend} from '../../../common/SendContext'
+
+export const useDeleteAmountWhenZeroed = () => {
+  const navigation = useNavigation<TxHistoryRouteNavigation>()
+
+  const {amountChanged, amountRemoved, selectedTokenId} = useSend()
+
+  const wallet = useSelectedWallet()
+  const tokenInfo = useTokenInfo({wallet, tokenId: selectedTokenId})
+
+  return (inputAmount: string) => {
+    console.error('useDeleteAmountWhenZeroed', asQuantity(inputAmount), Quantities.isZero(asQuantity(inputAmount)))
```

Comment:
~asQuantiity~

Reply:
any time you use `asQuantity`, see if you can move it earlier in the logic

Reply:
lifted up